proto/warden/warden/v1beta2/events.proto

```diff
@@ -0,0 +1,129 @@
+syntax = "proto3";
+package warden.warden.v1beta2;
+
+import "warden/warden/v1beta2/key.proto";
+
+option go_package = "github.com/warden-protocol/wardenprotocol/warden/x/warden/types/v1beta2";
+
+// EventCreateSpace is emitted on Space creation
+message EventCreateSpace {
+  // space_id associated with the space.
+  uint64 space_id = 1;
+
+  // creator is the account address of the space creator.
+  string creator = 2;
+
+  // owners_count is the count of the space owners.
+  uint64 owners_count = 3;
```

Comment:
it's fine tracking this number, but then I expect it to be tracked every time this count changes as well (so in AddSpaceOwner and RemoveSpaceOwner) :)